roles/kubernetes/node/templates/kubelet.kubeadm.env.v1beta1.j2

```diff
@@ -0,0 +1,102 @@
+### Upstream source https://github.com/kubernetes/release/blob/master/debian/xenial/kubeadm/channel/stable/etc/systemd/system/kubelet.service.d/
```

Comment:
duplicate ，why not use kubelet.kubeadm.env?  

Reply:
The environment configuration of kubelet-config takes precedence over kubelet-kubeadm.env.
 I believe you should understand

Reply:
Can you please be more specific where is duplication? Do you get the logic of this changes correctly?

I will try to describe changes more detailed here. 
Move flags that are deprecated and will be removed from kubelet kubelet.kubeadm.env file to kubelet config, and add another template of kubelet env file that will be installed if version of kubernetes more than 1.13.0. Thats why kubelet.kubeadm.env.v1beta1.j2 still needed, but if you look more carefully you can see that there is removed flags that are now in kubeletConfiguration kind in config file kubelet.kubeadm.env

> The environment configuration of kubelet-config takes precedence over kubelet-kubeadm.env.
I believe you should understand

Not sure what you mean here, you mean that flags defined in kubelet.kubeadm.env have more precedence or that kubelet config file have more precedence?

Reply:
If both kubelet.kubeadm.env and kubeadm kubelet-config configmap are defined and have the same environment variables, kubelet.kubeadm.env has a higher priority.

Reply:
Yes i know that thats why in second template that will be installed if config is used removed the same flags, or you mean that i miss something?